pkg/util/tarcopy/tarcopy.go

```
@@ -0,0 +1,124 @@
+	return rtnChan, func(fi fs.FileInfo, file string) error {
+			// generate tar header
+			header, err := tar.FileInfoHeader(fi, file)
+			if err != nil {
+				return err
+			}
+
+			header.Name = filepath.Clean(strings.TrimPrefix(file, pathPrefix))
+			if header.Name == "" {
+				return nil
+			}
+			if strings.HasPrefix(header.Name, "/") {
```

golangci-lint (1.62.2)

S1017: should replace this `if` statement with an unconditional `strings.TrimPrefix`

(gosimple)